.microcks/overlays.yaml

```diff
@@ -0,0 +1,204 @@
+              prev: https://api.example.com/stations?page=1
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="origin")].example
+    description: Remove the example from get /trips - origin parameter.
+    remove: true
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="origin")]
+    description: Add Microcks examples for get /trips - origin parameter
+    update:
+      examples:
+        trips:
+          value: 'b2e783e1-c824-4d63-b37a-d8d698862f1d'
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="destination")].example
+    remove: true
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="destination")]
+    update:
+      examples:
+        trips:
+          value: '2024-02-01T09:00:00Z'
+
+  - target: $.paths["/trips"].get.responses["200"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.paths["/trips"].get.responses["200"].content["application/json"]
+    update:
+      examples:
+        trips:
+          summary: A list of available train trips
+          value:
+            data:
+              - id: ea399ba1-6d95-433f-92d1-83f67b775594
+                origin: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+                destination: b2e783e1-c824-4d63-b37a-d8d698862f1d
+                departure_time: '2024-02-01T10:00:00Z'
+                arrival_time: '2024-02-01T16:00:00Z'
+                price: 50
+                operator: Deutsche Bahn
+                bicycles_allowed: true
+                dogs_allowed: true
+              - id: 4d67459c-af07-40bb-bb12-178dbb88e09f
+                origin: b2e783e1-c824-4d63-b37a-d8d698862f1d
+                destination: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+                departure_time: '2024-02-01T12:00:00Z'
+                arrival_time: '2024-02-01T18:00:00Z'
+                price: 50
+                operator: SNCF
+                bicycles_allowed: true
+                dogs_allowed: true
+            links:
+              self: https://api.example.com/trips?origin=efdbb9d1-02c2-4bc3-afb7-6788d8782b1e&destination=b2e783e1-c824-4d63-b37a-d8d698862f1d&date=2024-02-01
+              next: https://api.example.com/trips?origin=efdbb9d1-02c2-4bc3-afb7-6788d8782b1e&destination=b2e783e1-c824-4d63-b37a-d8d698862f1d&date=2024-02-01&page=2
+
+  - target: $.paths["/bookings"].get.responses["200"].content["application/json"]["example"]
+    remove: true
+
+  - target: $.paths["/bookings"].get.responses["200"].content["application/json"]
+    update:
+      examples:
+        bookings:
+          summary: A list of bookings
+          value:
+            data:
+              - id: bfc5af2c-f477-43c4-8bdf-a00bdb939d65
+                trip_id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+                passenger_name: John Doe
+                has_bicycle: true
+                has_dog: true
+              - id: 1725ff48-ab45-4bb5-9d02-88745177dedb
+                trip_id: b2e783e1-c824-4d63-b37a-d8d698862f1d
```

Comment:
same here,  guess it should be a trip id, e.g. `4d67459c-af07-40bb-bb12-178dbb88e09f`?